src/overview/build-deploy.md

```diff
@@ -9,36 +9,36 @@ Every time you push to a live branch (a git branch with an active environment at
+> While most projects have a single `.platform.app.yaml` file, Platform.sh supports multiple applications in a single project.  It will scan the repository for `.platform.app.yaml` files in subdirectories and each directory containing one will be built as an independent application. A built application will not contain any directories above the one in which it is found. The system is smart enough not to rebuild applications that have already been built, so if you have multiple applications, only changed applications will be rebuilt and redeployed.
 
-The live environment is composed of multiple containers - both for your application(s) and for  the services it depends on. It also has a virtual network that connects them as well as a router that routes incoming requests to you application(s).
+The live environment is composed of multiple containers&mbdash;both for your application(s) and for the services it depends on. It also has a virtual network connecting them, as well as a router to route incoming requests to the appropriate application.
 
 Based on your application type the system will select one of our pre-built container images and run the following:
 
-1. First, any dependencies specified in the `.platform.app.yaml` file are installed.  Those include tools like Sass, Gulp, Drupal Console, or many others that you may need.  
+1. First, any dependencies specified in the `.platform.app.yaml` file are installed.  Those include tools like Sass, Gulp, Drupal Console, or any others that you may need.  
 
-2. Second, depending on the “build flavor” specified in the configuration file we run a series of standard commands. The default for PHP containers, for example, is simply to run `composer install`.
+2. Then, depending on the “build flavor” specified in the configuration file, we run a series of standard commands. The default for PHP containers, for example, is simply to run `composer install`.
 
-3. Finally, run the “build hook” from the configuration file.  The build hook is simply one or more shell commands that you write to finish creating your production code base.  That could be compiling Sass files, running a Gulp or Grunt script, rearranging files on disk, compiling an application in a compiled language, or whatever else you want.  Note that at this point all you have access to is the file system; there are no services or other databases available.
+3. Finally, we run the “build hook” from the configuration file.  The build hook comprises one or more shell commands that you write to finish creating your production code base.  That could be compiling Sass files, running a Gulp or Grunt script, rearranging files on disk, compiling an application in a compiled language, or whatever else you want.  Note that at this point, all you are able to access is the file system; there are no services or other databases available.
```

Comment:
The final comma is extraneous.

Reply:
Technically, I'm being lax on the comma here. I could go with "Note that, at this point, all you are..." to set off the non-essential clause, but since both phrases serve as an introductory clause, it's not totally necessary.

Reply:
I would prefer both commas then, as the core sentence is "all you are able to..."; Both "Note that" and "at this point" are additions, so let's either go with both commas or none.

Reply:
Fixed.